spec/examples.txt

```diff
@@ -0,0 +1,14 @@
+example_id                                                      | status | run_time        |
```

Comment:
Remove this file. I don't think this was auto-generated, but if it was it also should be in gitignore